descriptors/ModuleDescriptor-template.json

```diff
@@ -144,6 +153,21 @@
         }
       ]
     },
+    {
+      "id": "source-record-storage-cleanup",
```

Comment:
maybe we should move delete records endpoint to records interface, what do you think?